### ISSUE TITLE
Copy slick files to build file in default gulplette

### ISSUE THREAD

**Issue by michaeldippold** [webspec-design/wordpress-elixir#2]
It's trying to load the files from build/css (the gif directly from that folder and the fonts from a /fonts folder,  matching the structure of the files in the node_module)

**Comment by michaeldippold** [comment]
Added this code to fix the problem in tasks/main.js:

  ```
hook_after_copy: function(mix, args) {

  	mix.copy('node_modules/slick-carousel/slick/ajax-loader.gif', 'build/css');
  	mix.copy('node_modules/slick-carousel/slick/fonts', 'build/css/fonts');

  },
```

**Comment by ethanclevenger91** [comment]
Isn't this a bit opinionated?

**Comment by webspecnick** [comment]
This whole project is opinionated. I'm currently thinking of including this code, but commented out, like how we do it in the [main.scss](https://github.com/webspec-design/wordpress-elixir/blob/master/sass/main.scss#L65). It'll be there for those who use it a lot.

**Comment by webspecnick** [comment]
Fixed with #4
